package.json

```diff
@@ -131,5 +134,20 @@
       "<rootDir>/jestMockSetup.js"
     ],
     "cacheDirectory": ".jest/cache"
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "$root$": true,
```

Comment:
Why allow in root? Shouldn't we use false as default so we have to manually authorize scripts?

Reply:
The idea was to allow our `postinstall` script to run, as we need to patch our packages with nodejs-compatible shims

But allowscripts won't allow it, even with `$root$: true`, so I removed it.

Reply:
Ok. Thanks!